<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrifierGUI</class>
 <widget class="QMainWindow" name="ErrifierGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ErrifierGUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="errifier.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="txt_file">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type the File Location or use the Browse Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="watch">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add To Watch List</string>
    </property>
   </widget>
   <widget class="QToolButton" name="browse">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_friendly">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter a Friendly Name(Preferably the service name)</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>561</width>
      <height>411</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="errifier.qrc"/>
 </resources>
 <connections/>
</ui>
